network-libp2p/src/network.rs

```diff
@@ -182,11 +183,10 @@ impl PubsubId<PeerId> for GossipsubId<PeerId> {
 pub struct Network {
     /// The local ID that is used to identify our peer
     local_peer_id: PeerId,
-    /// This hash map mantains an association between PeerIds and PeerContact:
+    /// This hash map maintains an association between PeerIds and PeerContact:
     /// If the peer is interesting, i.e.: it provides services that are interested to us,
-    ///  we store a some value with the peer contact itself
-    /// Otherwise,
-    ///  we just store a None value
+    /// we store a some value with the peer contact itself
```

Comment:
This comment is outdated, since we are storing the PeerContact and not an Option